macos py38 CI failing

`import matplotlib` is failing when it imports `PIL`

```python

E       ImportError: dlopen(/usr/local/miniconda/envs/xarray-tests/lib/python3.8/site-packages/PIL/_imaging.cpython-38-darwin.so, 2): Library not loaded: @rpath/libwebp.7.dylib
E         Referenced from: /usr/local/miniconda/envs/xarray-tests/lib/libtiff.5.dylib
E         Reason: Incompatible library version: libtiff.5.dylib requires version 9.0.0 or later, but libwebp.7.dylib provides version 8.0.0

/usr/local/miniconda/envs/xarray-tests/lib/python3.8/site-packages/PIL/Image.py:69: ImportError
```

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -32,16 +32,15 @@ jobs:
   steps:
   - template: ci/azure/unit-tests.yml
 
-# excluded while waiting for https://github.com/conda-forge/libwebp-feedstock/issues/26
-# - job: MacOSX
-#   strategy:
-#     matrix:
-#       py38:
-#         conda_env: py38
-#   pool:
-#     vmImage: 'macOS-10.15'
-#   steps:
-#   - template: ci/azure/unit-tests.yml
+- job: MacOSX
+  strategy:
+    matrix:
+      py38:
+        conda_env: py38
+  pool:
+    vmImage: 'macOS-10.15'
+  steps:
+  - template: ci/azure/unit-tests.yml
 
 - job: Windows
   strategy:

